docs/marketplace-required-csv-annotations.md

```diff
@@ -0,0 +1,79 @@
+# Marketplace Required CSV Annotations
+
+An operator's CSV must contain the following annotations for it to be displayed properly within the Marketplace UI:
```

Comment:
@galletti94 could you review the annotations mentioned below? Could you also confirm that marketplace is not the `links` annotation?